ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -939,6 +939,46 @@ router bgp {{ router_bgp.as }}
       route import match-failure action discard
 {%         endif %}
 {%     endif %}
+{# address family ipv4 path_selection activation #}
```

Comment:
The config here does not match the input data model.

- Redistribute routes are not in the input model
- `networks`, `bgp.missing_policy` and `bgp.additional_path` is in the model but not covered here
- Peer group route-maps are here but not in the model.

I don't know which is correct, so please verify on EOS and make sure to implement it consistently.